dominodatalab/python-domino
# [RM-1594] - python domino release-1.1.1

### Link to JIRA

[RM-1594](https://dominodatalab.atlassian.net/browse/RM-1594)

### What issue does this pull request solve?
bug fix from 1.1.0 where python would have issues running when installed from pypl as opposed to installed from direct whl.

### What is the solution?
include main python packing in installation build.

### Testing

1. Install latest version of domino 
- `$ pip install dominodatalab`
2. run python repl
- `$ python`
4. check python version
- `>> from domino import domino`
- `>> domino. __version__` # should return latest version 1.1.1
5. call some domino api.

_e.g. "I ran an upgrade from 4.2 to 4.6"._

- [ ] Unit test(s)

### Pull Request Reminders

- [x] Has relevant documentation been updated?
- [x] Does the code follow [Python Style Guide] (https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html)
- [x] Update the [changelog](https://github.com/dominodatalab/python-domino/blob/master/CHANGELOG.md)
- [x] Are the existing unit tests still passing?
- [ ] Have new unit tests been added to cover any changes to the code?
- [x] Has the JIRA ticket(s) been linked above?

### References (optional)
